### PR TITLE
Hide Dropdown When Search Input is Empty

### PR DESCRIPTION
**Pull request by nummi**
Addresses: https://github.com/ember-learn/guides-app/issues/65

### CODE REVIEW

File: addon/components/search-input.js [hunk 1]
```diff
@@ -27,6 +28,9 @@ export default Component.extend({
   // _projectVersion: alias('_projectService.standardisedVersion'),
   _results: A(),
   _focused: false,
+  _dropdownVisible: computed('_focused', 'value', function() {
+    return get(this, '_focused') && !!get(this, 'value').length;
```
Comment (sivakumar-kailasam) [inline addon/components/search-input.js]: `isPresent` might be more verbose here.

**Review by sivakumar-kailasam — CHANGES_REQUESTED** [review]

File: addon/components/search-input.js (hunk 1 above)
Comment (nummi) [reply addon/components/search-input.js]: @sivakumar-kailasam  Like this?

``` javascript
_dropdownVisible: computed('_focused', 'value', function() {
  return get(this, '_focused') && isPresent(get(this, 'value'));
})
```

Comment (sivakumar-kailasam) [reply addon/components/search-input.js]: Yes

Comment (nummi) [reply addon/components/search-input.js]: Change made.

**Review by sivakumar-kailasam — APPROVED** [review]
Thanks @nummi

**Pull request merged**